src/rcc.rs

```diff
@@ -37,12 +47,36 @@ pub struct Rcc {
 
 #[allow(unused)]
 const HSI: u32 = 8_000_000; // Hz
+#[allow(dead_code)] // Only used for ADC, but no configuration for ADC clock yet.
+const HSI14: u32 = 14_000_000; // Hz - ADC clock.
+// Can't do this anymore... #[cfg(any(feature = "stm32f042", feature = "stm32f072"))]
+const HSI48: u32 = 48_000_000; // Hz - (available on STM32F04x, STM32F07x and STM32F09x devices only)
+
+pub enum SysClkSource {
+    HSI = 0b00,
+    HSE = 0b01,
+    PLL = 0b10,
+    HSI48 = 0b11,
+}
+
+pub enum PllSource {
```

Comment:
This as well

Reply:
Same here. I will remove pub from both of these.